modules/collections/src/test/java/org/apache/tamaya/collections/CollectionAdvancedTests.java

```diff
@@ -47,12 +45,12 @@
     public void testCustomSeparator(){
         Configuration config = Configuration.current();
         List<String> items = config.get("sep-list", new TypeLiteral<List<String>>(){});
-        assertNotNull(items);
-        assertFalse(items.isEmpty());
-        assertEquals(3, items.size());
-        assertEquals("a,b,c", items.get(0));
-        assertEquals("d,e,f", items.get(1));
-        assertEquals("g,h,i", items.get(2));
+        assertThat(items).isNotNull();
+        assertThat(items.isEmpty()).isFalse();
```

Comment:
assertThat.isEmpty() seems shorter

Reply:
I agree, I would never the less merge the change. We can still improve it later.